spec/classes/service_spec.rb

```diff
@@ -8,7 +8,8 @@
   context 'default' do
     let :facts do
       {
-        os: { family: 'Unsupported' }
+        os: { family: 'Unsupported' },
+        operatingsystem: 'Unsupported'
```

Comment:
There are other spec tests where this should probably be reinstated, but good enough for now.